tests/test_core.py

```diff
@@ -7179,6 +7179,235 @@ def test_mallinfo(self):
   def test_wrap_malloc(self):
     self.do_run(open(path_from_root('tests', 'wrap_malloc.cpp')).read(), 'OK.')
 
+
+  def test_proxyfs(self):
```

Comment:
the test should be in `test_other.py` as it does not need to run in all different optimization modes